Terminal.Gui/ConsoleDrivers/WindowsDriver.cs

```diff
@@ -1760,9 +1751,9 @@ internal override void End ()
 		WinConsole?.Cleanup ();
 		WinConsole = null;
 
-		if (!RunningUnitTests && (_isWindowsTerminal || _parentProcessName == "devenv")) {
+		if (!RunningUnitTests && _isWindowsTerminal) {
 			// Disable alternative screen buffer.
-			Console.Out.Write (EscSeqUtils.CSI_RestoreCursorAndActivateAltBufferWithBackscroll);
+			Console.Out.Write (EscSeqUtils.CSI_RestoreAltBufferWithBackscroll);
```

Comment:
The name is correct but the value must be 1049 and `CSI_RestoreCursorAndActivateAltBufferWithBackscroll` should be 1047. The same for `CSI_ActivateAltBufferNoBackscroll`.

Reply:
This issue reappear because of this. The status bar stay visible at the bottom after exit.

![Captura de ecrã 2023-10-20 190548](https://github.com/gui-cs/Terminal.Gui/assets/13117724/2f97e65c-f878-49e7-8203-622ebbf6d2df)

Reply:
Sometimes I just do dumb things. Fixed.

Reply:
It's possible to invert the values on these fields? Because the current names are inverted.

Reply:
> It's possible to invert the values on these fields? Because the current names are inverted.

I'm not sure what you mean. I just went back and re-read the documentation and I believe these are accurate:

```cs
	/// <summary>
	/// ESC [ ? 1047 h - Activate xterm alternative buffer (no backscroll)
	/// </summary>
	/// <remarks>
	/// From https://invisible-island.net/xterm/ctlseqs/ctlseqs.html#h3-Functions-using-CSI-_-ordered-by-the-final-character_s_
	/// Use Alternate Screen Buffer, xterm. 
	/// </remarks>
	public static readonly string CSI_ActivateAltBufferNoBackscroll = CSI + "?1047h";

	/// <summary>
	/// ESC [ ? 1047 l - Restore xterm working buffer (with backscroll)
	/// </summary>
	/// <remarks>
	/// From https://invisible-island.net/xterm/ctlseqs/ctlseqs.html#h3-Functions-using-CSI-_-ordered-by-the-final-character_s_
	/// Use Normal Screen Buffer, xterm.  Clear the screen first if in the Alternate Screen Buffer.
	/// </remarks>
	public static readonly string CSI_RestoreAltBufferWithBackscroll = CSI + "?1047l";

	/// <summary>
	/// ESC [ ? 1049 h - Save cursor position and activate xterm alternative buffer (no backscroll)
	/// </summary>
	/// <remarks>
	/// From https://invisible-island.net/xterm/ctlseqs/ctlseqs.html#h3-Functions-using-CSI-_-ordered-by-the-final-character_s_
	/// Save cursor as in DECSC, xterm. After saving the cursor, switch to the Alternate Screen Buffer,
	/// clearing it first.
	/// This control combines the effects of the 1047 and 1048 modes.
	/// Use this with terminfo-based applications rather than the 47 mode.
	/// </remarks>
	public static readonly string CSI_SaveCursorAndActivateAltBufferNoBackscroll = CSI + "?1049h";

	/// <summary>
	/// ESC [ ? 1049 l - Restore cursor position and restore xterm working buffer (with backscroll)
	/// </summary>
	/// <remarks>
	/// From https://invisible-island.net/xterm/ctlseqs/ctlseqs.html#h3-Functions-using-CSI-_-ordered-by-the-final-character_s_
	/// Use Normal Screen Buffer and restore cursor as in DECRC, xterm.
	/// resource.This combines the effects of the 1047 and 1048  modes.
	/// </remarks>
	public static readonly string CSI_RestoreCursorAndActivateAltBufferWithBackscroll = CSI + "?1049l";
```

Reply:
Ahh, I now see the typo:

`CSI_RestoreCursorAndActivateAltBufferWithBackscroll` should be `CSI_RestoreCursorAndRestoreAltBufferWithBackscroll`

Reply:
I knew you'd get there :-)